src/pro/plugins/live-search/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { registerBlockVariation } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import { addFilter } from '@wordpress/hooks';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { assign } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Edit from './edit.js';
+
+const addAttribute = ( props ) => {
+	if ( 'core/search' === props.name ) {
+		props.attributes = assign( props.attributes, {
```

Comment:
We should use some attribute less generic, maybe something with Otter prefix to avoid some other plugin using the same attribute.

Reply:
On the top of that, instead of a PostTypes attribute, we should have some otterSearchQuery attribute as an object, where this goes to the `post_type` item, in case in future we want to add more query attributes, like limiting the number, order and so on. An example being the query loop:
`
"query":{"perPage":3,"pages":0,"offset":0,"postType":"page","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false,"parents":[]}`